scripts/performance/argument_parser.py

```diff
@@ -257,6 +257,30 @@ def parse_cli_args():
         required=False,
         default=None,  # NOTE: DO NOT SET DEFAULT TO FALSE, IT WILL BE OVERRIDDEN BY THE RECOMMENDED MODEL CONFIGS
     )
+    parser.add_argument(
+        "-fsdp",
+        "--use_mcore_fsdp",
+        help="Enable Mcore FSDP. Disabled by default",
+        action="store_true",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "-rl",
+        "--recompute_layers",
+        type=int,
+        help="Number of layers to recompute. Defaults to 0",
```

Comment:
Fix to `Number of Transformer layers to recompute, where all the intermediate activations of a Transformer layer are computed. Defaults to 0`